### DISCUSSION TITLE
Dont Destroy on Load Warning

### DISCUSSION THREAD

**Discussion by Zilk** [FirstGearGames/FishNet#498] [Feature Request]
Add this to NetworkObject.TryStartDeactivation to avoid DontDestroyOnLoad warning if object is placed inside another transform.

![image](https://github.com/FirstGearGames/FishNet/assets/7714186/98b1df16-2ac0-480c-b90e-88fd5da634cf)
![image](https://github.com/FirstGearGames/FishNet/assets/7714186/159e4074-4751-4df1-8e72-11babfec6270)

**Comment by FirstGearGames** [comment]
This was added.
